docs/classes.rst

```diff
@@ -298,7 +298,7 @@ inheritance relationship. This is reflected in Python:
 
     >>> p = example.pet_store()
     >>> type(p)  # `Dog` instance behind `Pet` pointer
-    Pet          # no pointer upcasting for regular non-polymorphic types
+    Pet          # no pointer downcasting for regular non-polymorphic types
```

Comment:
Good catch.  (I though I might have been responsible for that -- for some reason I've always has trouble remembering which way is "up" and "down" in an inheritance tree.  But nope, it looks like @dean0x7d wrote that, so I guess I'm not alone in confusing the directions :) ).

Reply:
It's definitely confusing to me too - I looked it up on Wikipedia before making this change, just to be sure :-)